uninstall.php

```diff
@@ -28,6 +28,24 @@
  * and to ensure only the site owner can perform this action.
  */
 if ( Constants::is_true( 'WC_REMOVE_ALL_DATA' ) ) {
+	/**
+	 * Load core packages autoloader.
+	 */
+	if ( version_compare( PHP_VERSION, '5.6.0', '>=' ) ) {
+		if ( ! class_exists( 'Automattic\WooCommerce\Admin\Install' ) ) {
```

Comment:
Why is this if statement needed? I didn't check but I believe that before calling uninstall.php, WP will load the plugin, so it should be safe to assume that the autoloader has been initialized. Or am I missing something? If we are worried that for some reason it is not possible to initialize the autoloader and then call `\Automattic\WooCommerce\Admin\Install::init()` (or more likely `drop_tables()` as Claudio mentioned below), maybe it is better to check if the method exist before calling it? My concern is that if we return here, the code that handles the removal of the rest of WooCommerce will not run.

Reply:
Ignore the part where I said that WP loads the plugin before calling uninstall.php. I just checked and that is not the case.